apps/classroomio-com/src/app.html

```diff
@@ -1,113 +1,134 @@
-  <!-- <link rel="stylesheet" href="global.css"> -->
-  <link rel="manifest" href="/manifest.json" crossorigin="use-credentials" />
-  <link rel="icon" type="image/png" href="/favicon.ico" />
-  <link rel="icon" type="image/png" sizes="32x32" href="/logo-32.png" />
+    <script
+      async
+      src="https://analytics.umami.is/script.js"
+      data-website-id="efd70f1f-1f2e-4513-a1dd-ba7826fec138"
+      data-domains="classroomio.com"
+    ></script>
 
-  <script async src="https://analytics.umami.is/script.js" data-website-id="efd70f1f-1f2e-4513-a1dd-ba7826fec138"
-    data-domains="classroomio.com"></script>
+    <script type="text/javascript">
+      window.$crisp = [];
+      window.CRISP_WEBSITE_ID = '69176b47-5f63-42cc-bb9b-68d4927b6276';
+      (function () {
+        d = document;
+        s = d.createElement('script');
+        s.src = 'https://client.crisp.chat/l.js';
+        s.async = 1;
+        d.getElementsByTagName('head')[0].appendChild(s);
+      })();
+    </script>
 
-  <script type="text/javascript">
-    window.$crisp = [];
-    window.CRISP_WEBSITE_ID = '69176b47-5f63-42cc-bb9b-68d4927b6276';
-    (function () {
-      d = document;
-      s = d.createElement('script');
-      s.src = 'https://client.crisp.chat/l.js';
-      s.async = 1;
-      d.getElementsByTagName('head')[0].appendChild(s);
-    })();
-  </script>
-
-  <script type="text/javascript">
-    (function (C, A, L) {
-      let p = function (a, ar) {
-        a.q.push(ar);
-      };
-      let d = C.document;
-      C.Cal =
-        C.Cal ||
-        function () {
-          let cal = C.Cal;
-          let ar = arguments;
-          if (!cal.loaded) {
-            cal.ns = {};
-            cal.q = cal.q || [];
-            d.head.appendChild(d.createElement('script')).src = A;
-            cal.loaded = true;
-          }
-          if (ar[0] === L) {
-            const api = function () {
-              p(api, arguments);
-            };
-            const namespace = ar[1];
-            api.q = api.q || [];
-            typeof namespace === 'string' ? (cal.ns[namespace] = api) && p(api, ar) : p(cal, ar);
-            return;
-          }
-          p(cal, ar);
+    <script type="text/javascript">
+      (function (C, A, L) {
+        let p = function (a, ar) {
+          a.q.push(ar);
         };
-    })(window, 'https://app.cal.com/embed/embed.js', 'init');
-    Cal('init', { origin: 'https://cal.com' });
-
-    Cal('ui', {
-      theme: 'light',
-      styles: { branding: { brandColor: '#1D4EE2' } },
-      hideEventTypeDetails: false,
-      layout: 'month_view'
-    });
-  </script>
-  %sveltekit.head%
-</head>
+        let d = C.document;
+        C.Cal =
+          C.Cal ||
+          function () {
+            let cal = C.Cal;
+            let ar = arguments;
+            if (!cal.loaded) {
+              cal.ns = {};
+              cal.q = cal.q || [];
```

Comment:
Something weird is going on with your formating in this file.

I don't see any actual code change.

Reply:
i just went through the file and compared it to app.html on the main branch asides mu prettier correcting the indentation, the other change is the meta content of the "twitter:description", i've replaced it with the one on main, i don't remember ever coming to this file but i've changed the content back to the one on main, apologies.

Reply:
Thank you